components/resources/BUILD.gn

```diff
@@ -151,9 +151,13 @@ repack("resources") {
 
   if (!is_android && !is_ios) {
     deps += [
+      "//brave/browser/resources/settings/email_aliases_page:generated_resources",
```

Comment:
I don't know where the shortcuts_page dep below came from or how it got here because this is a layering violation. You can't include `//brave/browser` from `//brave/components`. This should be failing gn check, if it's not I need to check the rules.

Reply:
On my machine it did not fail `npm run gn_check`.

Reply:
Hmm this is probably my fault - I'll go and tidy up the shortcuts page. Weird `gn_check` didn't catch this

Reply:
cc @arthuredelstein @bridiver PTAL at https://github.com/brave/brave-core/pull/28875

Reply:
fixed -- thanks for the example @fallaciousreasoning